instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -318,12 +391,17 @@ def __init__(
         request_hook: typing.Optional[RequestHook] = None,
         response_hook: typing.Optional[ResponseHook] = None,
     ):
+        _OpenTelemetrySemanticConventionStability._initialize()
+        self.semconv = _OpenTelemetrySemanticConventionStability._get_opentelemetry_stability_opt_in_mode(
```

Comment:
```suggestion
        self._semconv = _OpenTelemetrySemanticConventionStability._get_opentelemetry_stability_opt_in_mode(
```

Reply:
I'll use `_sem_conv_opt_in_mode` like the other ones